README.md

```diff
@@ -22,7 +22,7 @@ You can find the full documentation for GC Design System Components on [https://
 
 - Clone the repo git clone `https://github.com/cds-snc/gcds-components`.
 - Run `npm install` to install all Node.js dependencies.
-- Run `npm install –workspaces` to install all Node.js dependencies for the web, React and Angular packages.
+- Run `npm install -–workspaces` to install all Node.js dependencies for the web, React and Angular packages.
 - Change into the `/packages/web` directory.
 - Run `npm run build` to compile web components.
 
```

Comment:
Should the `s` at the end of `web packages` be removed in the sentence below?
Otherwise, people might think that there is more than one web package.
Can you also capitalize Angular and React, please?